src/R/Components/Impl/PackageManager/Implementation/View/PackageList.xaml

```diff
@@ -153,6 +153,7 @@
             <!-- the style of the ListBoxItem when the item is a package. -->
             <Style x:Key="PackageItemStyle" TargetType="{x:Type ListBoxItem}" BasedOn="{StaticResource FillingListBoxItemStyle}">
                 <EventSetter Event="MouseDoubleClick" Handler="List_MouseDoubleClick" />
+                <Setter Property="AutomationProperties.Name" Value="{Binding AccessibleDescription}" />
```

Comment:
You can split `AccessibleDescription` between `AutomationProperties.Name` and `AutomationProperties.HelpText`. This will allow user to skip the details if he doesn't want to hear them.